<robot name="test_robot">
  <link name="base_link">
    <visual>
      <geometry>
        <box size="0.32 0.16 0.12"/>
      </geometry>
      <origin rpy="0 0 0" xyz="0.1 0 0.085"/>
      <material name="white">
        <color rgba="1 1 1 1"/>
      </material>
    </visual>
  </link>
  
  <link name="lwheel">
    <visual>
      <geometry>
        <cylinder length="0.008" radius="0.046"/>
      </geometry>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <material name="black">
        <color rgba="0 0 0 1"/>
      </material>
    </visual>
  </link>
  
  <joint name="base_to_lwheel" type="continuous">
    <axis xyz="0 0 1"/>
    <limit effort="1000.0" lower="0.0" upper="0.548" velocity="0.5"/>
    <parent link="base_link"/>
    <child link="lwheel"/>
    <origin xyz="0 0.115 0.046" rpy="1.5708 0 0"/>
  </joint>
  
  <link name="rwheel">
    <visual>
      <geometry>
        <cylinder length="0.008" radius="0.046"/>
      </geometry>
      <origin rpy="0 0 0" xyz="0 0 0"/>
      <material name="black">
        <color rgba="0 0 0 1"/>
      </material>
    </visual>
  </link>
  
  <joint name="base_to_rwheel" type="continuous">
    <axis xyz="0 0 1"/>
    <limit effort="1000.0" lower="0.0" upper="0.548" velocity="0.5"/>
    <parent link="base_link"/>
    <child link="rwheel"/>
    <origin xyz="0 -0.115 0.046" rpy="-1.5708 0 0"/>
  </joint>  
</robot>